src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -1026,6 +1024,10 @@ namespace OpenMS
     uis_threshold_peak_area_ = param_.getValue("uis_threshold_peak_area");
     scoring_model_ = param_.getValue("scoring_model");
 
+    sn_win_len_ = (double)param_.getValue("TransitionGroupPicker:PeakPickerMRM:sn_win_len");
```

Comment:
why the extra cast?
There are operator()'s for DataValue which do this implicitly.